### PR TITLE
The Interactive-list For to-do list app

### PR DESCRIPTION
**Pull request by ifeanyi305**
Functionality to the to-do list application to make it interactive.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -2,48 +2,25 @@
 import _ from 'lodash';
 /* eslint-enable */
```
Comment (elmar8287) [inline src/index.js:3]: - [ ] Please, consider removing all linter disables. Let the linter does it's direct job - find bugs in the early steps of your project 👍

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi, @ifeanyi305.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights

- All required implementations are working correctly 👍 
 
## Required Changes ♻️

- [ ] Check the comments under the review.

## Optional suggestions

- I suggest to you give the name for the commits with a Capital letter to make it looks more professional 👍 
- Please, also consider fixing the broken link to the MIT.md file 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (ifeanyi305) [reply src/index.js:3]: Hi, @elmar8287, Thanks for the feedback
I used the "eslint-disable" for the Lodash only. besides linters can't pass if I don't disable it

**Review by AshakaE — APPROVED** [review]
Hi @ifeanyi305 , 

### Highlights ✨ 

- Git flow ✅ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (elmar8287) [reply src/index.js:3]: @ifeanyi305 in this case you can keep this disabled and go on. So, I see you have got approved the PR. Good luck 👍

**Comment by ifeanyi305** [comment]
Thank you

**Pull request merged**